gno.land/cmd/gnoland/testdata/event.txtar

```diff
@@ -0,0 +1,26 @@
+loadpkg gno.land/p/demo/ufmt
+
+# start a new node
+gnoland start
+
+gnokey maketx addpkg -pkgdir $WORK -pkgpath gno.land/r/demo/ee -gas-fee 1000000ugnot -gas-wanted 2000000 -broadcast -chainid=tendermint_test test1
+stdout OK!
+
+gnokey maketx call -pkgpath gno.land/r/demo/ee -func Hello -gas-fee 1000000ugnot -gas-wanted 2000000 -broadcast -chainid=tendermint_test test1
```

Comment:
Well, since the `timestamp` and `GAS USED` are not a fixed value, it was ambiguous to add the output in the txtar files. If partial matching is possible, I'll add it.

Reply:
hmmmmm.. I thought this would work, but it's failing at the `EVENTS` part. I'll have to find another way.

```plain
gnokey maketx call -pkgpath gno.land/r/demo/ee -func Hello -gas-fee 1000000ugnot -gas-wanted 2000000 -broadcast -chainid=tendermint_test test1
stdout OK!
stdout 'GAS WANTED: 2000000'
stdout 'GAS USED:   [0-9]+'
stdout 'EVENTS: [type: sender, pkgPath: gno.land/r/demo/ee, fn: Hello timestamp: [0-9]+, attributes: [key1: value1 key2: value2 key3: value3]]'
```

I think I should add a function that converts the Event vector into a string to output the event, but I'm not sure. 🤔🤔

```go
type ResponseBase struct {
	Error  Error
	Data   []byte
	Events []Event

	Log  string // nondeterministic
	Info string // nondeterministic
}

// ...

func ExecSignAndBroadcast (...) {
 // ...
 io.Println("EVENTS:	", bres.DeliverTx.Events.String()) // <- something like this
}
```

Reply:
@notJoon @zivkovicmilos consider the `stdout` call to work like a `grep`: as long as the last output contains the regex pattern, it will succeed. There is no need to match the entire line or output.

Reply:
> There is no need to match the entire line or output.

Now I get it. then if written extremely, using just `'EVENTS: ' `for stdout also would be sufficient. thanks!